LICENSE-MIT

```diff
@@ -1,21 +1,19 @@
-MIT License
+## License
 
-Copyright (c) 2020 Electric Coin Company
+Copyright 2017-2021 CikeQiu.
+Copyright 2020-2021 The Electric Coin Company.
 
-Permission is hereby granted, free of charge, to any person obtaining a copy
-of this software and associated documentation files (the "Software"), to deal
-in the Software without restriction, including without limitation the rights
-to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
-copies of the Software, and to permit persons to whom the Software is
-furnished to do so, subject to the following conditions:
+Licensed under either of
```

Comment:
This is the wrong file to make these changes in (it is `LICENSE-MIT`, and needs to remain as such). These changes should be in `README.md`, and then if you want it to also show up in GitHub, add a `COPYING.md` file with this content.

Reply:
See e.g. https://github.com/zcash/librustzcash/pull/449.

Reply:
gotcha! Thanks !